src/platform/silabs/wifi/rs911x/platform/efx32_ncp_host.c

```diff
@@ -42,18 +40,20 @@
 #include "sl_board_control.h"
 #endif // SL_BOARD_NAME
 
+#include "sl_si91x_ncp_utility.h"
+#include <platform/silabs/wifi/wf200/platform/spi_multiplex.h>

```

Comment:
These don't need to be seperate - can we keep them with the rest?

Reply:
Updated as per comment